Unity-Technologies/ScriptableRenderPipeline
# Fix local config package path

**DONT FORGET TO ADD A CHANGELOG**

---
### Purpose of this PR
Fix the path of the local config package that the wizard send to the package manager api.
Before the missing "/" was leading to not adding the local package to the manifest.

---
### Testing status

**Manual Tests**: What did you do?
Tested locally:
- Attached rider to check the new path value.
- Changed Shaderconfig.cs.hlsl to check that it was indeed affecting the project.
